resume.yaml

```diff
@@ -1,121 +0,0 @@
-name: Sean Gatewood
```

Comment:
I might rename the file `SeanGatewoodResume.pdf` or something more descriptive than `resume.pdf` since I bet recruiters get a lot `resume.pdf`s